tibber_aws/s3.py

```diff
@@ -49,6 +49,10 @@ async def load_data(self, key, if_unmodified_since=None):
         data, _, state = await self.load_data_metadata(key, if_unmodified_since)
         return data, state
 
+    async def delete_file(self, key):
+        await self.init_client_if_required()
+        self._client.delete_object(Bucket=self._bucket_name, Key=key)
```

Comment:
I would assume it should be
```suggestion
        await self._client.delete_object(Bucket=self._bucket_name, Key=key)
```

Reply:
Done